analytics-core/src/main/java/com/segment/analytics/http/SegmentService.java

```diff
@@ -6,6 +6,6 @@
 
 /** REST interface for the Segment API. */
 public interface SegmentService {
-  @POST("/v1/import")
+  @POST("")
```

Comment:
Didn't realize this was allowed - is there something in Retrofit's documentation that suggests this is ok/not ok?